Skill/app/src/main/java/com/skilltradiez/skilltraderz/Skill.java

```diff
@@ -278,19 +278,29 @@ public void setImages(List<Image> images) {
     public void setImage(Image image, int pos) {
         this.images.set(pos, image);
     }
+
+    // Better Specific Method
     public Image getImage(int pos) {
         return images.get(pos);
     }
 
+    @Deprecated
     public Image getImage() {
         return images.size() == 0 ? new NullImage() : images.get(0);
     }
+
+    public List<Image> getImages() {
```

Comment:
problem with this is if you do getImages() then modify the returned list, notifyDB isn't called.